app/services/integrations/ltsa_parcel_map_bc.rb

```diff
@@ -26,7 +23,7 @@ def historic_site_by_pid(pid:)
     query = "where=HISTORIC_SITE_IND='Y' AND PARCEL_DESCRIPTION='#{pid}'&returnGeometry=true&outFields=*"
     response = @client.get("#{ENV["GEO_LTSA_PARCELMAP_REST_URL"]}#{HISTORIC_SERVICE}/query?f=json&#{query}")
     #assuem if there is a parcel description match not to use ltsa geometry matching
-    return response if response.success? && response.body.dig("features").length > 0
+    return parse_attributes_from_response(response) if response.success? && response.body.dig("features").length > 0
```

Comment:
what happens if features is nil

Reply:
if the pid fials to be found (returns nil), it goes onto look fo rthe geometry of the pid, then suppies that geometry for a intersection with historic sites to see if there is any overlap